lib/src/cmaf/ham/types/model/Track.ts

```diff
@@ -51,6 +51,11 @@ type AudioTrack = Track & {
  */
 type TextTrack = Track;
 
+type FrameRate = {
```

Comment:
Maybe , i would add leave this at the end of the file so as to follow convention